core/src/main/java/org/elasticsearch/common/xcontent/ToXContent.java

```diff
@@ -26,6 +26,8 @@
 
 /**
  * An interface allowing to transfer an object to "XContent" using an {@link XContentBuilder}.
+ * The output may not be valid alone without a valid ancestor, or without starting and ending an
```

Comment:
What about "The output may or may not be a value object. Objects implementing {@link ToXContentObject} output a valid value but those that don't may or may not require emitting a startObject and an endObject."